modules/launcher/src/test/java/org/mule/runtime/module/launcher/application/DefaultApplicationFactoryTestCase.java

```diff
@@ -71,6 +92,8 @@ public void createsApplication() throws Exception
         assertThat(application.getDescriptor(), is(descriptor));
         assertThat(application.getArtifactName(), is(APP_NAME));
         assertThat(application.getResourceFiles(), is(resourceFiles));
+        // Descriptor doesn't contain the core application plugin installed for this app
+        assertThat(application.getDescriptor().getPlugins().contains(appPlugin), is(false));
```

Comment:
The descriptor is provided is created on this test, there is no reason why it would change if there is a plugin in the container